src/components/editor/Bindings/Selector.tsx

```diff
@@ -64,7 +72,7 @@ function BindingSelector() {
     ];
 
     useEffect(() => {
-        if (!isEmpty(currentCollection)) setSelectionModel(currentCollection);
+        if (currentCollection) setSelectionModel([currentCollection]);
```

Comment:
`currentCollection` is either a `string` or `null`, therefore evaluating the constant directly should suffice.